portal/src/main/resources/application.conf

```diff
@@ -45,4 +45,14 @@ database {
     defaultFileVersion = "v0.0.1"
     defaultFileVersion = ${?CMWL_DEFAULT_FILE_VERSION}
   }
+
+  mongo {
+    user = "mongoUser"
+    password = "password"
+    host = "127.0.0.1"
+    port = 27017
```

Comment:
Don't we need to store all eternal resources parameters directly in the conf file? 2) maybe we need to have a short description of how to tune our mongo's in Readme file? Don't you add any script to docker-composer?

Reply:
Good point, I will add it. No, didn't add any script.

Reply:
@OstapStrashevskii Add explanation how to integrate with MongoDB to KB. Plz check, am I did it right?